chains/evm/contracts/pools/USDC/USDCTokenPoolProxy.sol

```diff
@@ -0,0 +1,378 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {IPoolV1} from "../../interfaces/IPool.sol";
+import {IRouter} from "../../interfaces/IRouter.sol";
+import {ITypeAndVersion} from "@chainlink/contracts/src/v0.8/shared/interfaces/ITypeAndVersion.sol";
+
+import {ERC165CheckerReverting} from "../../libraries/ERC165CheckerReverting.sol";
+import {Pool} from "../../libraries/Pool.sol";
+import {USDCSourcePoolDataCodec} from "../../libraries/USDCSourcePoolDataCodec.sol";
+import {USDCTokenPool} from "./USDCTokenPool.sol";
+
+import {Ownable2StepMsgSender} from "@chainlink/contracts/src/v0.8/shared/access/Ownable2StepMsgSender.sol";
```

Comment:
nit: first party, not third

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1134/commits/3ebbcc46301b035096dac695582297dd1f783bfe#diff-510793de9f7a7a1b0dd14ec9abdc694fe11b9eb710c18b19be666c59bc459840L6